src/Altinn.App.Api/Controllers/AuthorizationController.cs

```diff
@@ -123,4 +82,74 @@ public async Task<IActionResult> ValidateSelectedParty(int userId, int partyId)
             return StatusCode(500, $"Something went wrong when trying to validate party {partyId} for user {userId}");
         }
     }
+
+    /// <summary>
+    /// Fetches roles for current party.
+    /// </summary>
+    /// <returns>List of roles for the current user and party.</returns>
+    [Authorize]
+    [HttpGet("{org}/{app}/api/authorization/roles")]
+    public async Task<IActionResult> GetRolesForCurrentParty()
```

Comment:
We need to add annotations so that the openapi generation can figure out what our endpoints returns.

Should look something like this:
```suggestion
    [ProducesResponseType(typeof(IEnumerable<Roles), StatusCodes.Status200OK)]
    [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status400BadRequest)]
    public async Task<IActionResult<IEnumerable<Roles>> GetRolesForCurrentParty()
```